cargo-pgx/src/command/install.rs

```diff
@@ -101,7 +101,7 @@ pub(crate) fn install_extension(
     base_directory: Option<PathBuf>,
     features: &clap_cargo::Features,
 ) -> eyre::Result<()> {
-    let base_directory = base_directory.unwrap_or("/".into());
+    let base_directory = base_directory.unwrap_or_else(|| PathBuf::from("/"));
```

Comment:
Note that this allocates, so it doesn't break the "actually `clippy::or_fun_call` is wrong here" criteria I mentioned before. Unsure if it matters for perf tho'. However, I think this lint should be allowed, after this.